File: AndroidApps/MyFragment/app/src/main/java/com/jinco/myfragment/MainActivity.java
package com.jinco.myfragment;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    Button btn1, btn2;
    Fragment box;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                FirstFragment ff = (TextView) getFragmentManager().findFragmentById(R.id.frament_1);
            }
        });

        btn2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

            }
        });
    }
}

File: AndroidApps/RecordKeeperPractice_2/app/src/main/java/com/jinco/recordkeeperpractice_2/MainActivity.kt
package com.jinco.recordkeeperpractice_2

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import com.jinco.recordkeeperpractice_2.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding:ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
        setSpinner()
        binding.btnSubmit.setOnClickListener { onClick() }
    }

    private fun setSpinner() {
        val wha: Array<String> = arrayOf("wh", "th")
        val spinnerAdapter =
            ArrayAdapter(this, android.R.layout.simple_spinner_dropdown_item, wha)
        binding.spJob.adapter = spinnerAdapter
    }

    private fun onClick() {
        val message = Message(
            binding.etContactName.text.toString(),
            binding.etContactNumber.text.toString(),
            binding.etDisplayName.text.toString(),
            binding.spJob.selectedItem.toString(),
            binding.cbImmediate.isChecked
        )

        val intent = Intent(this, PreviewActivity::class.java)
        intent.putExtra("message", message)
        startActivity(intent)
    }
}